client/components/PopularSearches.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import styled from 'styled-components';
+
+const PopularContainer = styled.div`
+  border: 1px black;
+`;
+
+const CardContainer = styled.div`
```

Comment:
Is this going to be used?

Reply:
That might be an outdated, that styled component was deleted